### PR TITLE
Reducers actions

### PR DESCRIPTION
**Pull request by KaskMIL**
# In this new Pull Request I added the following features to the project:
- Added a <ins>**Redux directory**</ins> that contains the following elements:
   - <ins>**Books directory**</ins>  that contains the file BooksRedux.js with the Action types, Initial state, bookReducer and Action creators, and a test file to test the reducer.
   - <ins>**Categories directory**</ins> that contains the file CategoriesRedux.js with the Action types, Initial state, bookReducer and Action creators, and a test file to test the reducer.
   - <ins>**configStore.js file**</ins> that initializes the store and combines the reducers.

### CODE REVIEW

File: src/redux/Books/BooksRedux.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// Actions
+const ADDBOOK = 'ADD_BOOK';
+const DELETEBOOK = 'DELETE_BOOK';
+
```
Comment (Gambit142) [inline src/redux/Books/BooksRedux.js]: - [ ] Kindly make the value of each action type follow the [duck pattern](https://github.com/erikras/ducks-modular-redux). 

#### Hint: `const ADD_BOOK = bookstore/books/ADD_BOOK`

File: src/redux/Books/BooksReduxTest.test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  const book = { title: 'El Alquimista', author: 'Cohelo', id: 2 };
+  const final = [{ title: 'Foundation', author: 'Asimov', id: 1 }, { title: 'the crow', author: 'Edgard Allan Poe', id: 3 }];
+  expect(actions.default(initial, actions.deleteBook(book.id))).toEqual(final);
+});
+
+test('Unkonwn action', () => {
+  const initial = [{ title: 'Foundation', author: 'Asimov', id: 1 }, { title: 'the crow', author: 'Edgard Allan Poe', id: 3 }];
+  const action = () => ({ type: 'unknown' });
+  expect(actions.default(initial, action)).toEqual(initial);
+});
```
Comment (Gambit142) [inline src/redux/Books/BooksReduxTest.test.js:21]: - [ ] Great work writing unit tests for your reducers. That is great effort and professionalism. 👏

File: src/redux/Categories/CategoriesRedux.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// Actions
+const CHECKSTATUS = 'CHECK_STATUS';
+
```
Comment (Gambit142) [inline src/redux/Categories/CategoriesRedux.js]: - [ ] Kindly make the value of each action type follow the [duck pattern](https://github.com/erikras/ducks-modular-redux). 

#### Hint: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @KaskMIL 👋, 

Good job so far!

## Highlights
- `Redux toolkit` is present in `package.json` ✔️ 
- The structure of the files follows **_duck pattern_** ✔️
- Configured `Redux` store correctly ✔️
- Created `Action types`, `Action creators` and `reducers` correctly ✔️   

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by KaskMIL** [comment]
Hi @Gambit142,
Thanks for your review, I'll implement the changes you suggest
Have a great day!
cheers

**Review by DeliceLydia — APPROVED** [review]
## Hi @KaskMIL, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KaskMIL** [comment]
Hi @DeliceLydia,
Thanks for your review!
Have a great day, cheers!

**Comment by Gambit142** [comment]
Hi @KaskMIL, great work done. 👍

**Pull request merged**